Cargo.lock

```diff
@@ -1016,8 +1016,8 @@ dependencies = [
  "escape8259",
  "hex",
  "indexmap 2.1.0",
- "ink_env 5.0.0-rc (registry+https://github.com/rust-lang/crates.io-index)",
- "ink_metadata 5.0.0-rc (registry+https://github.com/rust-lang/crates.io-index)",
+ "ink_env 5.0.0-rc",
+ "ink_metadata 5.0.0-rc",
```

Comment:
We should first release these two packages, then release a new `contract-transcode`, then update to that version. Also will need to release and update `contract-build` at the same time, though that has no `ink` deps itself.

We should probably add that to the release process.